### PR TITLE
Input updates part 2

### PR DESCRIPTION
**Pull request by StephenHodgson**
## Overview

Finishes PR from https://github.com/XRTK/XRTK-Core/pull/193

### CODE REVIEW

File: XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/Collections/BaseObjectCollection.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ public abstract class BaseObjectCollection : MonoBehaviour
         /// <summary>
         /// List of objects with generated data on the object.
         /// </summary>
-        protected List<ObjectCollectionNode> NodeList { get; } = new List<ObjectCollectionNode>();
+        public List<ObjectCollectionNode> NodeList { get; } = new List<ObjectCollectionNode>();
```
Comment (StephenHodgson) [inline XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/Collections/BaseObjectCollection.cs]: Should return a `IReadonlyLIst<ObjectCollectionNode>`

Might wanna pull this out into it's own pr with the undo ports as well.

File: XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/BaseFocusHandler.cs [hunk 2]
```diff
@@ -33,12 +34,12 @@ public virtual bool FocusEnabled
...
         /// Does this object currently have focus by any <see cref="IMixedRealityPointer"/>?
         /// </summary>
-        public virtual bool HasFocus => FocusEnabled && Focusers.Count > 0;
+        public virtual bool HasFocus => FocusEnabled && ActivePointers.Count > 0;
 
         /// <summary>
         /// The list of <see cref="IMixedRealityPointer"/>s that are currently focused on this <see cref="GameObject"/>
         /// </summary>
-        public List<IMixedRealityPointer> Focusers { get; } = new List<IMixedRealityPointer>(0);
+        public List<IMixedRealityPointer> ActivePointers { get; } = new List<IMixedRealityPointer>(0);
```
Comment (StephenHodgson) [inline XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/BaseFocusHandler.cs]: This should be readonly

File: XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/ControllerMappings/DefaultLuminControllerMappingProfile.asset [hunk 3]
```diff
@@ -13,7 +13,6 @@ MonoBehaviour:
   m_Name: DefaultLuminControllerMappingProfile
   m_EditorClassIdentifier: 
   isCustomProfile: 0
-  parentProfile: {fileID: 11400000, guid: f83e1d143c5a46dba5e70f497a7a996d, type: 2}
```
Comment (SimonDarksideJ) [inline XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/ControllerMappings/DefaultLuminControllerMappingProfile.asset]: Was this breaking of parenting intentional?

Comment (StephenHodgson) [reply XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/ControllerMappings/DefaultLuminControllerMappingProfile.asset]: Yes, this property is no longer serialized.
